pkg/datapath/loader/loader.go

```diff
@@ -199,7 +199,7 @@ func hostRewrites(cfg *datapath.LocalNodeConfiguration, ep datapath.Endpoint, if
 		opts["SECCTX_FROM_IPCACHE"] = uint64(secctxFromIpcacheDisabled)
 	}
 
-	opts["NATIVE_DEV_IFINDEX"] = uint64(ifIndex)
+	opts["nodeport_dev_ifindex"] = uint64(ifIndex)
```

Comment:
Can't we remove this line instead of renaming now that we have the constants in LoadAndAssign?